client-api/src/main/kotlin/org/ostelco/prime/client/api/TopupConfiguration.kt

```diff
@@ -23,4 +23,6 @@ class TopupConfiguration {
     fun setAuthenticationCachePolicy(spec: String) {
         this.authenticationCachePolicy = CacheBuilderSpec.parse(spec)
     }
+    var pseudonymEndpoint = ""
```

Comment:
Should be in the config file I think - in `prime/config/config.yaml` to be specific. And if not set in config then the app. should fail at startup due to invalid config.

Reply:
This is already in that `config.yaml`. Where should we test this, in `prime` or `client-api` module ?

Reply:
@kmmatwork  What do you mean by fail, Should I thrown an exception from `TopupModule.kt` ?

Reply:
Yes, add a `@NotNull` annotation and it will stop prime form starting if not configured correctly.

Not sure about testing as we currently don't have tests for configs in the api module. But the config is simple and it might be a bit of overkill to have tests for it?